sql/municipios.sql

```diff
@@ -5,5578 +5,5581 @@ CREATE TABLE municipios(
     longitude FLOAT(8) NOT NULL,
     capital BOOLEAN NOT NULL,
     codigo_uf INT NOT NULL,
-    PRIMARY KEY (codigo_ibge),
-    FOREIGN KEY (codigo_uf) REFERENCES estados (codigo_uf)
+    siafi_id INT NOT NULL,
+    ddd INT NOT NULL,
+    fuso_horario VARCHAR(50),
+    PRIMARY KEY (codigo_ibge)
+    FOREIGN KEY (codigo_uf) REFERENCES estados (codigo_uf),
```

Comment:
Como é a última linha antes do fechamento do parênteses, essa vírgula estava quebrando o sql ao tentar criar a tabela no mysql/mariadb:

```suggestion
    FOREIGN KEY (codigo_uf) REFERENCES estados (codigo_uf)
```